lib/ooxml_parser/xlsx_parser/xlsx_data/view_model/workbook/worksheet/table_part/extension_list/extension/conditional_formattings/conditional_formatting/conditional_formatting_rule/color_scale.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module OoxmlParser
+  # Class for `colorScale` data
+  class ColorScale < OOXMLDocumentObject
+    # @return [Array, ConditionalFormatValueObject] list of values
+    attr_reader :values
+    # @return [Array, Color] list of colors
```

Comment:
`# @return [Array<OoxmlColor>] list of colors`

and same syntax for array in attr_reader before